src/card.py

```diff
@@ -1,3 +1,10 @@
+import json
+#Implementation of a standard Poker card, with ability to output to JSON
 class Card:
-    def __init__(self):
-        print("Card constructor")
+    def __init__(self, rank, suit):
+        self.rank = {"rank": rank}
```

Comment:
I don't think these need to be dictionaries. They could just be the value itself, e.g.

```
    self.rank = rank
    self.suit = suit
```

Making them a dict will probably make things harder down the  road. Also, in Python we typically use an underscore to denote data members of a class, e.g.

```
    self._rank = rank
```

This is true in JavaScript as well. While not entirely necessary, it might make it a little easier to denote member variables from other types of variables.